README.md

```diff
@@ -14,15 +14,20 @@ These dependencies are:
 In most cases, running `./install.sh` should be sufficient to install all dependencies for emirge and mirgecom.
 
 `./install.sh` takes several arguments:
-- `--prefix=DIR`: Install conda in `DIR` instead of the default directory (`$HOME/miniforge3`).
+- `--conda-prefix=DIR`: Install conda in `DIR` instead of the default directory (`./miniforge3`).
+- `--env-name=NAME`: Create conda environment named `NAME` instead of the default (dgfem).
+- `--install-prefix=DIR`: Install mirgecom and git pip packages to `DIR` instead of the default (./).
+- `--branch=NAME`: Install the `NAME`d branch of mirgecom instead of the default branch (master).
+- `--conda-pkgs=FILE`: Install additional conda packages from list specified in `FILE`.
+- `--pip-pkgs=FILE`: Install additional pip packages from list specified in `FILE`.
 - `--modules`: Install a modules.zip file that contains a copy of all python packages that are installed through git (see below for details).
 
 ## Testing the installation
 
 Testing can be done by:
 
 ```bash
-$ cd mirgecom/examples; python wave-eager.py
+$ cd mirgecom/examples; ./run_examples.sh ./
```

Comment:
```suggestion
$ mirgecom/examples/run_examples.sh mirgecom/examples/
```

Reply:
yup - far better.  even better would be if the run_examples.sh would default to ./ (which it currently doesn't, I don't think)